collection-scripts/gather_ceph_resources

```diff
@@ -269,7 +268,34 @@ for ns in $namespaces; do
             fi
             find "${COMMAND_OUTPUT_DIR}" -name "rbd_vol_and_snap_info_*.part" -print0 | xargs -0 cat >>"${COMMAND_OUTPUT_FILE}"
             find "${COMMAND_OUTPUT_DIR}" -name "rbd_vol_and_snap_info_*.part" -print0 | xargs -0 rm -f
-        done
+
+            rados_namespaces=$(timeout 60 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd -p $bp namespace ls")
+            for rns in $rados_namespaces; do
```

Comment:
add radosnamespace to below code as well

```
ceph_resources=()
ceph_resources+=(cephblockpools)
ceph_resources+=(cephfilesystems)
ceph_resources+=(cephclient)
ceph_resources+=(cephfilesystemsubvolumegroups)
```

Reply:
+1

Reply:
`oc adm inspect --dest-dir="${CEPH_COLLECTION_PATH}" --all-namespaces "${resource}" ` we already collect all namespace for these resources, I don't think we need to add anything seperately.

Reply:
...correct me if I'm wrong, but the `--all-namespaces` here refers to k8s namespaces, and not RADOS namespaces. We're asking you to add `ceph_resources+=(cephblockpoolradosnamespaces)` to gather the rns CRs.

Reply:
oh is it, sorry, I will check and update accordingly.